README.md

```diff
@@ -25,311 +25,435 @@ produced it. The image tests are reproduced below, with each preceded by its
 ## Gallery
 
 ```
-$ degenerate all apply save
+$ degenerate rotate-color:g:0.5 all apply save
+
 ```
-![all apply save](images/all%20apply%20save.png)
+![rotate-color:g:0.5 all apply save
+](images/00.png)
```

Comment:
There's an extra newline here that should be there. The string should be trimmed after reading it from disk.